locale/en/auth.ftl

```diff
@@ -286,7 +286,11 @@ subscriptionUpdatePayment-plaintext = To prevent any interruption to your servic
 
 # Variables:
 #  $supportUrl (String) - Link to https://support.mozilla.org/kb/im-having-problems-my-firefox-account
-support-message-2 = For more info, visit { -brand-mozilla } Support: { $supportUrl }.
+support-message-3 = For more info, visit <a data-l10n-name="supportLink">{ -brand-mozilla } Support</a>}.
```

Comment:
Going to fix this, as it's creating a bunch of sync errors in Pontoon

```
Jul 2 22:11:40	mozilla-pontoon app/worker.1 [2025-07-02 22:11:40,107: ERROR/ForkPoolWorker-2] [mozilla-accounts:auth.ftl, pt-PT] Update failed: Unbalanced closing brace in TextElement. after message subscriptionUpdatePayment-plaintext at line 289
```

Reply:
https://github.com/mozilla/fxa-content-server-l10n/commit/80e3d96041062b291c4c0b52d4e2d9c7c788e706